pkg/system/phase4_configuring.go

```diff
@@ -747,6 +791,22 @@ func (r *Reconciler) ReconcileDefaultBackingStore() error {
 	return nil
 }
 
+func (r *Reconciler) prepareDefaultPVC() error {
+	// We need to create a PVC with specturm scale storageclass. nsfs-local is just for testing.
```

Comment:
I guess we should fix this before merge?

Reply:
Yes.